utility/install_prereq.py

```diff
@@ -1,19 +1,19 @@
-from ceph.utils import setup_repos
+from cli.cephadm.ansible import Ansible
 from cli.utilities.packages import Package, SubscriptionManager
-from cli.utilities.utils import get_custom_repo_url, os_major_version
-from utility.log import Log
-from utility.utils import fetch_build_artifacts
+from cli.utilities.utils import get_builds_by_rhbuild, get_custom_repo_url
 
-RHEL8_EPEL_RELESE_RPM = (
-    "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
-)
-RHCEPH5_TOOLS_REPO = "rhceph-5-tools-for-rhel-{}-x86_64-rpms"
-RHEL8_ANSIBLE_REPO = "ansible-2.9-for-rhel-8-x86_64-rpms"
+RHCS5_REPOS = {
+    "rhel-8": [
+        "rhceph-5-tools-for-rhel-8-x86_64-rpms",
```

Comment:
Need to add RHEL9 repo as well

subscription-manager repos --enable=rhceph-5-tools-for-rhel-9-x86_64-rpms

Reply:
+1

Reply:
Done